app/api/dashboardeditor.py

```
@@ -86,11 +86,11 @@ async def update_dashboard(dashboard_id: int, request: Request, db: Session = De
         }
 
         db.commit()
-        print(f"✅ Dashboard ID {dashboard_id} updated successfully.")
+       
         return {"message": "Dashboard updated successfully."}
 
     except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)